http_parser.c

```diff
@@ -472,6 +484,107 @@ static struct {
 
 int http_message_needs_eof(const http_parser *parser);
 
+#if defined(_MSC_VER) // SSE2 is baseline
+#include <intrin.h>
+#define __builtin_ctz _tzcnt_u32
+#elif defined(__SSE2__) && defined(__GNUC__)
+#include <emmintrin.h>
+#endif
+
+#ifndef USE_INTRISICS_CRLF
+# if defined(_MSC_VER) || \
+    (defined(__SSE2__) && defined(__GNUC__))
+#   define USE_INTRISICS_CRLF 1
+# endif
+#endif
+
+#if USE_INTRISICS_CRLF
+
+static uint32_t get_crlf_mask(const char* p) {
+  /* [ c, 0, 0, 0, 0, 0 .. 0 ] */
+  const __m128i vCR = _mm_set1_epi8(0x0d);
+
+  /* [ c, 0, 0, 0, 0, 0 .. 0 ] */
+  const __m128i vLF = _mm_set1_epi8(0x0a);
+
+  __m128i v1, v2;
+  v1 = _mm_load_si128((__m128i*)p);
+  v2 = _mm_cmpeq_epi8(vCR, v1);
+  v1 = _mm_cmpeq_epi8(vLF, v1);
+  v2 = _mm_or_si128(v1, v2);
+  return _mm_movemask_epi8(v2);
+}
+
+const char* find_crlf(const char* p, const char* data, size_t len) {
+  const char* lastp = MIN(data + len, HTTP_MAX_HEADER_SIZE + p);
+  uint32_t result = 0;
+
+  while ((uintptr_t)p & 15 && p <= lastp) {
+    if ( *p == CR || *p == LF ) {
```

Comment:
The extra spacing like this should be removed to match the style of the rest of the code.